_data/chinna98/chinna98.md

```diff
@@ -0,0 +1,7 @@
+---
+name: ChinnapaReddy Kanakanti 
+institution: Saveetha University 
+profile_pic: Chinna.png
```

Comment:
Hello @chinna98, this name does not match the image name that you have uploaded. Please make it so both have the same name.